src/components/AssetIconWithBadge.tsx

```diff
@@ -34,13 +34,16 @@ export const AssetIconWithBadge: React.FC<AssetIconWithBadgeProps> = ({
   children,
 }) => {
   const isNft = useMemo(() => {
-    return Object.values(transfersByType)
-      .flat()
-      .some(transfer => !!transfer.id)
+    return (
+      transfersByType &&
```

Comment:
Turbo nit but prefer wrapping this in a `Boolean()` cast to keep its previous boolean type vs. `boolean | undefined` now

Reply:
Actioned in https://github.com/shapeshift/web/pull/7004/commits/b39ea0eb699d2309beeb8cef365131bd0bba9bab